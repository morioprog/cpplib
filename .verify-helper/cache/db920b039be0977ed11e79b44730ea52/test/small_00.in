1 20
334853366452
2 0 1 196566914955
1 0 1 811054169756
3 0 1
0 0 1 -760100108583
3 0 1
3 0 1
2 0 1 543239771882
3 0 1
2 0 1 25846124544
0 0 1 -653713505276
3 0 1
2 0 1 113885385617
3 0 1
2 0 1 53614523289
0 0 1 -999485687906
1 0 1 -22471788279
2 0 1 -587358895864
1 0 1 530496693772
3 0 1
0 0 1 546527696824
