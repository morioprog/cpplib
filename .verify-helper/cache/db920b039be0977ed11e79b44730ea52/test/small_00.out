811054169756
-760100108583
-760100108583
-216860336701
-653713505276
-539828119659
530496693772
